homeassistant/components/influxdb.py

```diff
@@ -114,7 +114,8 @@ def influx_event_listener(event):
 
         for key, value in state.attributes.items():
             if key != 'unit_of_measurement':
-                json_body[0]['fields'][key] = value
+                if isinstance(value, (str, int, float)):
```

Comment:
I think that booleans should also be allowed. Have you tested and seen that not working?